src/ios/iOSAccessibilityUtil.ts

```diff
@@ -0,0 +1,22 @@
+/**
+* iOSAccessibilityUtil.ts
```

Comment:
Once again, the file name shouldn't contain iOS since that's redundant. It's already in a folder called iOS. Same with the class name it exports.